src/ApiCore/GapicClientTrait.php

```diff
@@ -95,117 +99,193 @@ private static function getGapicVersion(array $options)
      * @param array $options {
      *     An array of required and optional arguments.
      *
-     *     @type string $versionFile
```

Comment:
I think this is still used: https://github.com/michaelbausor/gax-php/blob/d7cdbdfaedaa4bd74b2306e9322a35b8b8d17e2c/src/ApiCore/GapicClientTrait.php#L84